tests/testlib/s2n_testlib.h

```diff
@@ -217,7 +217,10 @@ int s2n_test_cert_chain_and_key_new(struct s2n_cert_chain_and_key **chain_and_ke
  * cert with `p256` or `p384`
  * @param digest indicates the certificate signature digest of `sha256` or
  * `sha384`
-*/
+ * 
+ * @note OpenSSL FIPS module requires the key size for RSA cert to be bigger than or equal to 1024 bits.
+ * See https://github.com/aws/s2n-tls/issues/4651 for more information.
```

Comment:
```suggestion
 * @note The OpenSSL FIPS module requires the key size for RSA certs to be greater than or equal to 1024 bits.
 * See https://github.com/aws/s2n-tls/issues/4651 for more information.
```

Worth also mentioning Openssl-3, or nah? https://github.com/aws/s2n-tls/pull/4654#discussion_r1697715616

Reply:
I'd vote for no until we confirm this behavior. I'm not sure if anyone is using OpenSSL 3.x.